src/nevermined/AccessService.ts

```diff
@@ -25,26 +25,20 @@ export interface AccessProofTemplateParams {
 
 export class AccessService extends Instantiable implements ServicePlugin<ServiceAccess> {
     normal: AccessTemplate
-    proof?: AccessTemplate
 
     constructor(config: InstantiableConfig, normal: AccessTemplate) {
         super()
         this.setInstanceConfig(config)
         this.normal = normal
     }
 
-    // essential method is to select between two services
-    public select(main: MetaDataMain): ServicePlugin<ServiceAccess> {
-        return this.isDTP(main) ? this.proof : this.normal
-    }
-
     public async createService(
         publisher: Account,
         metadata: MetaData,
         assetRewards: AssetRewards,
         erc20TokenAddress: string
     ): Promise<ServiceAccess> {
-        return this.select(metadata.main).createService(
+        return this.normal.createService(
```

Comment:
not sure if `normal` us now the best name here

Reply:
This createService is not really needed either. it could be just implemented in the template. But I will it like this for now all the other plugins are using the same convention